libs/CMakeLists.txt

```diff
@@ -4,3 +4,7 @@ target_compile_definitions(Random123 INTERFACE R123_USE_MULHILO64_C99)
 
 add_library(h5xx INTERFACE)
 target_include_directories(h5xx SYSTEM INTERFACE ${CMAKE_CURRENT_SOURCE_DIR}/h5xx)
+
+if(STOKESIAN_DYNAMICS OR STOKESIAN_DYNAMICS_GPU)
```

Comment:
can't you add it unconditionally and decide later if sth. should be added to the build?

Reply:
The contents of libs/stokesian_dynamics are not supposed to know about Espresso specifics, so `STOKESIAN_DYNAMICS_GPU` and `STOKESIAN_DYNAMICS_GPU` can only be used here.